srcpkgs/pcsc-cyberjack/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'pcsc-cyberjack'
+pkgname=pcsc-cyberjack
+version=3.99.5
+revision=1
+_versext=SP15
+wrksrc=pcsc-cyberjack-${version}final.${_versext}
+build_style=gnu-configure
+configure_args="--enable-udev"
+hostmakedepends="pkg-config"
+makedepends="pcsclite-devel eudev-libudev-devel libusb-compat-devel"
+depends="pcsc-ccid libusb"
+short_desc="Linux drivers for cyberJack® RFID"
+maintainer="dataCobra <datacobra@thinkbot.de>"
+license="LGPL-2.1-or-later, GPL-2.0-or-later"
+homepage="https://www.reiner-sct.com/lang/en/support"
+#changelog=""
```

Comment:
```suggestion
```

Reply:
I checked once more for a changelog but couldn't find one. I removed this line as suggested. :+1: